site/layouts/partials/footer.html

```diff
@@ -1,56 +1,113 @@
-        </p>
-        <ul class="bd-footer-links ps-0 mb-0">
-          <li class="list-inline-item me-3 mb-3 mb-md-0 d-block d-md-inline-block"><span>Currently v{{ .Site.Params.current_version }}</span></li>
-          <li class="list-inline-item mt-3 mt-md-0"><a href="https://www.netlify.com">
-            <img src="https://www.netlify.com/img/global/badges/netlify-dark.svg" alt="Deploys by Netlify" width="89" height="40"/>
-          </a></li>
-        </ul>
+<footer class="footer bg-dark navbar-dark bd-footer">
+  <h2 class="visually-hidden">Boosted sitemap & information</h2>
+  <div class="container-xxl footer-nav">
+    <nav class="accordion accordion-dark" id="sitemapAccordion" aria-label="Boosted sitemap footer">
+      <div class="row">
+        <div class="footer-column col-md-6 col-lg-2">
+          <h3 class="accordion-header footer-heading" id="headingLinks">
+            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button"
+                    data-bs-toggle="collapse" data-bs-target="#collapseLinks" aria-expanded="true"
+                    aria-controls="collapseLinks">
+              Links
+            </button>
+            <span class="d-none d-md-flex">Links</span>
+          </h3>
+          <div id="collapseLinks" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+            <ul class="navbar-nav">
+              <li><a class="nav-link" href="/" aria-describedby="headingLinks">Home</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/"
+                     aria-describedby="headingLinks">Docs</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/examples/"
+                     aria-describedby="headingLinks">Examples</a></li>
+              <!-- Boosted mod: removed .Site.Params.themes and .Site.Params.blog -->
+            </ul>
+          </div>
+        </div>
+        <div class="footer-column col-md-6 col-lg-2">
+          <h3 class="accordion-header footer-heading" id="headingGuides">
+            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button"
+                    data-bs-toggle="collapse" data-bs-target="#collapseGuides" aria-expanded="true"
+                    aria-controls="collapseGuides">
+              Guides
+            </button>
+            <span class="d-none d-md-flex">Guides</span>
+          </h3>
+          <div id="collapseGuides" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+            <ul class="navbar-nav">
+
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/getting-started/"
+                     aria-describedby="headingGuides">Getting started</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/examples/starter-template/"
```

Comment:
We should update /examples/starter-template or remove this link. We can remove it now and create an issue to try to reintroduce a Boosted version for this example page if necessary.